anonymizer/base.py

```diff
@@ -72,7 +72,7 @@ def get_allowed_value(self, source, field):
 
         return retval
 
-    ### Public interace ##
+    # Public interace
```

Comment:
Maybe correct this `interface` typo while at it too. Unless this is a word I don't know about.